terraform/application/config/dv_review.tfvars.json

```diff
@@ -1,5 +1,7 @@
 {
   "namespace": "development",
   "deploy_azure_backing_services": false,
-  "enable_postgres_ssl": false
+  "enable_postgres_ssl": false,
+  "create_storage_account": true,
+  "storage_account_name": "s189d01ittmsrvsa"
```

Comment:
we usually add something descriptive to the sa name, e.g. s189d01ittmscsvrvsa

Reply:
This won't be specifically for CSVs, it will be the apps main/generic storage account. I've opted for a similarly generic name.

That being said, I'm unbiased about the infrastructure naming conventions.

What do you think?

Reply:
maybe using 'app' to identify it? so ittmsapprvsa